composer.lock

```diff
@@ -3345,7 +3345,8 @@
     "prefer-stable": false,
     "prefer-lowest": false,
     "platform": {
-        "php": ">=7.4 <=8.1"
+        "php": ">=7.4 <=8.1",
+        "ext-openssl": "*"
```

Comment:
Not sure why this was added automatically by composer... 🤔

Reply:
@st3iny didn't update the lock file with https://github.com/nextcloud/mail/pull/7741#pullrequestreview-1213095581 :)